ext/dom/dom.stub.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+class DOMNode {
+    /** @return DOMNode|false */
+    public function appendChild(DOMNode $newChild) {}
+
+    /** @return string|false */
+    public function C14N(bool $exclusive = false, bool $with_comments = false, ?array $xpath = NULL, ?array $ns_prefixes = NULL) {}
+
+    /** @return int|false */
+    public function C14NFile(string $uri, bool $exclusive = false, bool $with_comments = false, ?array $xpath = NULL, ?array $ns_prefixes = NULL) {}
+
+    /** @return DOMNode|false */
+    public function cloneNode(bool $recursive = false) {}
+
+    public function getLineNo() : int {}
```

Comment:
For consistency with other stubs, there should not be a space before the return type colon.